src/windows.rs

```diff
@@ -1,24 +1,39 @@
-use winapi::shared::minwindef;
-
 use std::os::windows::io::AsRawHandle;
 use std::process::Child;
-use std::ptr;
+
+mod windows {
+    use core::ffi::c_void;
+    pub(crate) use windows::Win32::{
+        Foundation::HANDLE,
+        System::{
+            Diagnostics::Debug::{ReadProcessMemory, WriteProcessMemory},
+            Threading::{
+                OpenProcess, PROCESS_CREATE_THREAD, PROCESS_QUERY_INFORMATION,
+                PROCESS_VM_OPERATION, PROCESS_VM_READ, PROCESS_VM_WRITE,
+            },
+        },
+    };
+
+    pub(crate) type DWORD = u32;
+    pub(crate) type LPCVOID = *const c_void;
+    pub(crate) type LPVOID = *mut c_void;
+}
 
 use super::{Architecture, CopyAddress, ProcessHandleExt, PutAddress, TryIntoProcessHandle};
 
 /// On Windows a `Pid` is a `DWORD`.
-pub type Pid = minwindef::DWORD;
+pub type Pid = windows::DWORD;
```

Comment:
If the `windows` crate doesn't define this directly, I think this should probably just be moved to `u32` instead of `DWORD`.

Reply:
Ok, this makes defining DWORD unnecessary, so I removed the type alias for it.